### PR TITLE
Create kubeflow-steering-committee-elections-2024.md

### PR DESCRIPTION
**Pull request by akgraner**
Updated the information, guidelines and timeline for this KSC 2024 election. 2 seats.

### CODE REVIEW

**Review by terrytangyuan — APPROVED** [review]
/lgtm
/approve

File: elections/eligible-candidates-and-voters-2024-ksc.md [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+- [vikas-saxena02](https://github.com/vikas-saxena02)
+- [anishasthana](https://github.com/anishasthana)
+- [kuizhiqing](https://github.com/kuizhiqing)
+- [gmfrasca](https://github.com/gmfrasca)
+- [jiridanek](https://github.com/jiridanek)
+- kannon92
+- shashank-iitbhu
+- dhirajsb
+- andersbennedsgaard
+- vsoch
```
Comment (terrytangyuan) [inline elections/eligible-candidates-and-voters-2024-ksc.md:62]: These are missing links?

Comment (akgraner) [reply elections/eligible-candidates-and-voters-2024-ksc.md]: I updated them today. There's only one I can't find now.

**Pull request merged**